### PR TITLE
Remove the hamburger menu for now

### PR DESCRIPTION
**Pull request by samgiles**
@wilsonpage r?

### CODE REVIEW

File: lib/views/header-bar.js [hunk 1]
```diff
@@ -84,22 +83,6 @@ class HeaderBarView extends React.Component {
...
-    return <TouchableOpacity
-      testId="hamburger-button"
-      style={styles.hamburger}
-      onPress={this.props.onHamburgerPressed.bind(this)}>
-      <Image
-        style={styles.hamburgerImage}
-        source={require('../images/header-bar-hamburger.png')}/>
-    </TouchableOpacity>
-  }
-
```
Comment (wilsonpage) [inline lib/views/header-bar.js:102]: Can you keep this function so that we can easily add it back later.

**Comment by wilsonpage** [comment]
A unit-test is failing. Can you `it.skip(` it?

**Comment by wilsonpage** [comment]
r+ after comments addressed

File: lib/views/header-bar.js (hunk 1 above)
Comment (samgiles) [inline lib/views/header-bar.js:102]: But it's in version control history! Not a fan of keeping dead code around fwiw.

**Pull request merged**